Source/SupportingFiles/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.0.1</string>
+	<string>0.1.0</string>
```

Comment:
These changes can also be reverted as part of my other comment

Reply:
Good to know, I will change it, thanks